source/sessions/tests/dirty-session-errors.yml

```diff
@@ -409,32 +198,3 @@ tests:
       collection:
         data:
           - {_id: 1}
-
-  - description: Dirty implicit session is discarded (non-cursor returning read)
```

Comment:
Why were these four tests removed?

* Clean explicit session is not discarded
* Clean implicit session is not discarded
* Dirty explicit session is discarded (non-bulk write)
* Dirty implicit session is discarded (non-bulk write)

Were they redundant? They seem unrelated to relaxing the prohibition on standalone deployments.

Reply:
hm I have no idea what happened here, restoring them now

Reply:
This was unintentional, I was copying from a branch that hadn't updated its local tests to include these more recent test additions. Fixed